### DISCUSSION TITLE
UK broadcasters are looking to set up their own IPTV service

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [ExperiencersInternational/tvsetup#25] [Announcements]
https://www.ispreview.co.uk/index.php/2023/09/britains-broadcasters-prep-new-free-live-tv-service-via-broadband.html

Saw this earlier browsing around.

I'm sorry that I'm pretty inactive here, but I generally haven't been watching TV on my TV, so I haven't been checking whether most channels are active or not (iirc they are still active).

If you have an issue or want to add a channel, please make sure to do so through the issue forms. Thank you!

**Comment by ExperiencersInternational** [comment]
https://www.ispreview.co.uk/index.php/2024/02/freelys-new-live-uk-broadband-tv-service-to-launch-q2-2024.html

Launching soon I guess

**Comment by ExperiencersInternational** [comment]
https://www.ispreview.co.uk/talk/threads/freely.41758/

I suspect the repo will be closing soon then, I honestly have no interest in maintaining it personally. I know people from abroad use this and I apologise to them in advance with this potentially being unavailable soon.

**Reply by dp247** [reply]
Rip

**Reply by ExperiencersInternational** [reply]
Looking into it further and it says new TVs? Not comfortable with only new TVs being compatible and not perfectly fine existing FVP ones

**Reply by dp247** [reply]
Yeah, saw something saying the likelihood of it coming to existing Freeview devices is slim.

Also, it's basically a fancy launcher for existing apps (iplayer, itvx etc), so kinda not sure what their market is 😂

**Reply by ExperiencersInternational** [reply]
I think it's frankly dumb, not gonna archive it for now but I haven't done much work as it is lately on the project. 

My TV is a 43in Toshiba Android TV, released in 2022 and it's been stuck on Android 11 security patch February 2022. It's dumb that Freely won't come to me by the sounds of it rn.

**Reply by morgogs** [reply]
A lot of manufacturers still have no support and no word on a timeline; I have an LG C2 running WebOS 23 and I'm not sure if it will ever get support. 

Love this project btw.

**Reply by tqeng** [reply]
This project is very awesome, I'm using it to get my poor aunt off paying for Sky services. She only watches Freeview channels, and was paying a ridiculous 120 quid to them. I set her up with a Firestick and all the public channels catch-up apps, but she misses having an EPG for liveTV. Freeview doesn't work on the Firestick so no EPG there, and I can't find her any affordable boxes that don't require the use of an aerial. And Freely doesn't work besides from those few shiny new Smart TVs! 

Thank you so much for this project, I'm not from the UK so I can't often help her out in-person so I sincerely hope it continues to function for a while! It's really helped us out.
